### DISCUSSION TITLE
NostrGit on Nostr

### DISCUSSION THREAD

**Discussion by cypherhoodlum** [NostrGit/NostrGit#86] [General]
npub16ygdhnz0slfpudnesl6nfr4hvv3gaznwa3u697t7heegk0tcwjnslg8n2r (https://no.str.bio/nostrgit)

I think the user should stay fairly quiet for now but created it for the future. Any zaps received will be redistributed to the team. Let's figure out the key management for the user at some point.

**Comment by PeerRich** [comment]
i wish there was some better account sharing mechanism yet for nostr

**Reply by cypherhoodlum** [reply]
yep, multisig on nostr.

**Comment by PeerRich** [comment]
lets set up a bitwarden maybe?

**Comment by PeerRich** [comment]
lets make a nostrgit@getalby.com over your personal one

**Reply by cypherhoodlum** [reply]
I wasn't sure we'd want to use Alby at all, but sure. How do you feel about using nostrgit@protonmail.com for the Alby account?

**Reply by cypherhoodlum** [reply]
I could create one or use my own email. Which do you prefer?

**Comment by PeerRich** [comment]
added a short link: https://no.str.bio/nostrgit

**Comment by davidabram** [comment]
It really bugs me that nostrgit npub doesn't start with vanity "npubn0strg7t..." :D

**Reply by PeerRich** [reply]
hahahah

**Reply by cypherhoodlum** [reply]
I'm on it :smile:

**Comment by cypherhoodlum** [comment]
We [decided](https://github.com/NostrGit/NostrGit/discussions/95) to not use Discussions anymore. Let's continue the discussion at #116  :heart:
